lib/atoms/a/a.component.js

```diff
@@ -0,0 +1,13 @@
+var lnAA = {
+    templateUrl: 'lnPatterns/atoms/a/template.html',
+    bindings: {
+        link: '@',
+        href: '@',
+        target: '@',
+        class: '@'
+    }
+};
+
+angular
```

Comment:
No, the variable must be defined before... If we define the component before the variable is defined, it results in an execution error :(